### PR TITLE
Adding watermark support to DebeziumIO transform

### PR DESCRIPTION
**Pull request by pabloem**
We had no watermark support before, which is a serious problem : /

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Mention the appropriate issue in your description (for example: `addresses #123`), if applicable. This will automatically add a link to the pull request in the issue. If you would like the issue to automatically close on merging the pull request, comment `fixes #<ISSUE NUMBER>` instead.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://beam.apache.org/contribute/get-started-contributing/#make-the-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)
[Go tests](https://github.com/apache/beam/actions?query=workflow%3A%22Go+tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI.

### CODE REVIEW

**Comment by pabloem** [comment]
r: @chamikaramj

**Comment by pabloem** [comment]
r: @chamikaramj would you be ablet o help out? : )

File: sdks/java/io/debezium/src/main/java/org/apache/beam/io/debezium/KafkaConnectUtils.java [hunk 1]
```diff
@@ -78,6 +79,18 @@ public static Schema.FieldType beamSchemaTypeFromKafkaType(
     }
   }
 
+  public static Instant debeziumRecordInstant(SourceRecord record) {
+    if (!record.valueSchema().type().equals(org.apache.kafka.connect.data.Schema.Type.STRUCT)
```
Comment (johnjcasey) [inline sdks/java/io/debezium/src/main/java/org/apache/beam/io/debezium/KafkaConnectUtils.java:83]: Can we add a test to verify this behavior as well?

**Review by johnjcasey — COMMENTED** [review]
Looks good, there is one more test we could add

**Comment by pabloem** [comment]
thanks for the review @johnjcasey ! I've added the test. Can you take a look please? : )

**Comment by pabloem** [comment]
broken suite is unrelated : )

**Comment by pabloem** [comment]
ptal : D

**Comment by pabloem** [comment]
@johnjcasey ptal : )

**Comment by johnjcasey** [comment]
LGTM

**Pull request merged**